### ISSUE TITLE
Error installing relays

### ISSUE THREAD

**Issue by leninmehedy** [hashgraph/solo#185]
When running `solo relay install` it throws error saying: `prompts.promptSelectNamespaceArg is not a function`

```
******************************* Solo *********************************************
Version                 : 0.22.1
Kubernetes Context      : kind-solo-e2e
Kubernetes Cluster      : kind-solo-e2e
Kubernetes Namespace    : solo
**********************************************************************************
(node:41459) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
✖ prompts.promptSelectNamespaceArg is not a function
◼ Prepare chart values
◼ Install JSON RPC Relay
*********************************** ERROR *****************************************
Error installing relays
***********************************************************************************

```

**Comment by leninmehedy** [comment]
relays is failing to run:
``` 
> start                                                                                                              │
│ > npx lerna exec --scope @hashgraph/json-rpc-server -- npm run start                                                 │
│                                                                                                                      │
│ lerna notice cli v8.0.1                                                                                              │
│ lerna info versioning independent                                                                                    │
│ lerna notice filter including "@hashgraph/json-rpc-server"                                                           │
│ lerna info filter [ '@hashgraph/json-rpc-server' ]                                                                   │
│ lerna info Executing command in 1 package: "npm run start"                                                           │
│                                                                                                                      │
│ > @hashgraph/json-rpc-server@0.39.1 start                                                                            │
│ > node dist/index.js                                                                                                 │
│                                                                                                                      │
│ Killed                                                                                                               │
│ npm ERR! Lifecycle script `start` failed with error:                                                                 │
│ npm ERR! Error: command failed                                                                                       │
│ npm ERR!   in workspace: @hashgraph/json-rpc-server@0.39.1                                                           │
│ npm ERR!   at location: /home/node/app/packages/server                                                               │
│ lerna ERR! npm run start exited 1 in '@hashgraph/json-rpc-server'                                                    │
│ lerna ERR! npm run start exited 1 in '@hashgraph/json-rpc-server'                                                    │
│ npm notice                                                                                                           │
│ npm notice New major version of npm available! 9.8.1 -> 10.5.0                                                       │
│ npm notice Changelog: <https://github.com/npm/cli/releases/tag/v10.5.0>                                              │
│ npm notice Run `npm install -g npm@10.5.0` to update!                                                                │
│ npm notice                                                                                                           │
│ Stream closed EOF for solo/relay-node0-hedera-json-rpc-relay-658665dbd4-9hxnl (hedera-json-rpc-relay)                │
```

**Comment by leninmehedy** [comment]
version `v0.43.0` works so I've set that as the default version in `solo`
